packages/rockets-next-web/src/app/(main)/users/UsersTable.tsx

```diff
@@ -0,0 +1,158 @@
+"use client";
+
+import type { RowProps } from "@concepta/react-material-ui/dist/components/Table/types";
+import { type FC, useCallback, useMemo } from "react";
+import Box from "@mui/material/Box";
+import IconButton from "@mui/material/IconButton";
+import { Table, createTableStyles } from "@concepta/react-material-ui";
+import {
+  TableContainer,
+  TableHead,
+  TableBody,
+  TableRow,
+  TableCell,
+} from "@mui/material";
+import useTheme from "@mui/material/styles/useTheme";
+import EditIcon from "@mui/icons-material/Edit";
+import ChevronRightIcon from "@mui/icons-material/ChevronRight";
+import DeleteIcon from "@mui/icons-material/Delete";
+
+import { headers } from "./tableConfig";
+import type { FormData, ActionType } from "./types";
+
+interface UsersTableProps {
+  isLoading?: boolean;
+  isEmptyStateVisible?: boolean;
+  data: FormData[];
+  onActionClick: ({
+    rowData,
+    action,
+  }: {
+    rowData: FormData;
+    action: ActionType;
+  }) => void;
+}
+
+const UsersTable: FC<UsersTableProps> = ({
+  isLoading,
+  isEmptyStateVisible,
+  data,
+  onActionClick,
+}) => {
+  const theme = useTheme();
+
+  const tableTheme = createTableStyles({
+    table: {
+      height: "100%",
+    },
+    root: {
+      display: "flex",
+      flexDirection: "column",
+      flex: 1,
+      overflow: "auto",
+    },
+    tableHeader: {
+      ...theme.typography.caption,
+      lineHeight: 1,
+      fontWeight: 500,
+      color: theme.palette.grey[500],
+    },
+    tableRow: {
+      backgroundColor: "#F9FAFB",
+      textTransform: "uppercase",
+    },
+    tableContainer: {
+      flex: 1,
+    },
+  });
+
```

Comment:
Create a Table component with this theming so we can reuse

Reply:
Done!